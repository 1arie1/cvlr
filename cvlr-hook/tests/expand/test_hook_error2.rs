use cvlr_hook::cvlr_hook_on_entry;

fn hook() {
    ();
}

fn hook2() {
    ();
}

#[cvlr_hook_on_entry(hook(), hook2())]
fn t1() {
    // hook inserted here
    println!("t1");
}

fn main() {}