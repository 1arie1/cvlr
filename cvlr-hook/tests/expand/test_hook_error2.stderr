error: Expected 1 argument
  --> tests/expand/test_hook_error2.rs:11:1
   |
11 | #[cvlr_hook_on_entry(hook(), hook2())]
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: this error originates in the attribute macro `cvlr_hook_on_entry` (in Nightly builds, run with -Z macro-backtrace for more info)
